package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pino-http-print",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
Can you please revert this?

Reply:
@mcollina Sure. Just so I understand correctly: You don't want the version number to be increased for this change?

Reply:
Exactly. We bump this on release.